<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
	<class>MenuClass</class>
	<widget class="QWidget" name="MenuClass">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>685</width>
				<height>400</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Menu</string>
		</property>
		<property name="styleSheet">
			<string notr="true"/>
		</property>
		<widget class="QLabel" name="label">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>681</width>
					<height>41</height>
				</rect>
			</property>
			<property name="font">
				<font>
					<pointsize>-1</pointsize>
				</font>
			</property>
			<property name="styleSheet">
				<string notr="true">
					background-color: transparent;
					color: #f53e2d;
					font-size: 28px;
					qproperty-alignment: AlignCenter;
				</string>
			</property>
			<property name="text">
				<string>Phòng Quản Trị</string>
			</property>
		</widget>
		<widget class="QToolButton" name="departmentButton">
			<property name="geometry">
				<rect>
					<x>10</x>
					<y>70</y>
					<width>151</width>
					<height>121</height>
				</rect>
			</property>
			<property name="font">
				<font>
					<pointsize>12</pointsize>
				</font>
			</property>
			<property name="styleSheet">
				<string notr="true">
					QToolButton {
					background-color: transparent;
					border: none;
					color: rgb(32, 137, 162);
					}

					QToolButton::hover {
					background-color: rgb(230, 237, 243);
					border: 1px solid rgb(180, 187, 194);
					border-radius: 8px;
					}
				</string>
			</property>
			<property name="text">
				<string>Phòng Ban</string>
			</property>
			<property name="icon">
				<iconset>
					<normaloff>../../Images/department.png</normaloff>../../Images/department.png
				</iconset>
			</property>
			<property name="iconSize">
				<size>
					<width>350</width>
					<height>350</height>
				</size>
			</property>
		</widget>
		<widget class="QToolButton" name="employeeButton">
			<property name="geometry">
				<rect>
					<x>180</x>
					<y>70</y>
					<width>151</width>
					<height>121</height>
				</rect>
			</property>
			<property name="font">
				<font>
					<pointsize>12</pointsize>
				</font>
			</property>
			<property name="styleSheet">
				<string notr="true">
					QToolButton {
					background-color: transparent;
					border: none;
					color: rgb(32, 137, 162);
					}

					QToolButton::hover {
					background-color: rgb(230, 237, 243);
					border: 1px solid rgb(180, 187, 194);
					border-radius: 8px;
					}
				</string>
			</property>
			<property name="text">
				<string>Nhân Viên</string>
			</property>
			<property name="icon">
				<iconset>
					<normaloff>../../Images/employ.png</normaloff>../../Images/employ.png
				</iconset>
			</property>
			<property name="iconSize">
				<size>
					<width>85</width>
					<height>85</height>
				</size>
			</property>
		</widget>
		<widget class="QToolButton" name="eventButton">
			<property name="geometry">
				<rect>
					<x>350</x>
					<y>70</y>
					<width>151</width>
					<height>121</height>
				</rect>
			</property>
			<property name="font">
				<font>
					<pointsize>12</pointsize>
				</font>
			</property>
			<property name="styleSheet">
				<string notr="true">
					QToolButton {
					background-color: transparent;
					border: none;
					color: rgb(32, 137, 162);
					}

					QToolButton::hover {
					background-color: rgb(230, 237, 243);
					border: 1px solid rgb(180, 187, 194);
					border-radius: 8px;
					}
				</string>
			</property>
			<property name="text">
				<string>
					Sự Kiện
					Kiểm Diện
				</string>
			</property>
			<property name="icon">
				<iconset>
					<normaloff>../../Images/event.png</normaloff>../../Images/event.png
				</iconset>
			</property>
			<property name="iconSize">
				<size>
					<width>75</width>
					<height>75</height>
				</size>
			</property>
		</widget>
		<widget class="QToolButton" name="exceptionButton">
			<property name="geometry">
				<rect>
					<x>520</x>
					<y>70</y>
					<width>151</width>
					<height>121</height>
				</rect>
			</property>
			<property name="font">
				<font>
					<pointsize>12</pointsize>
				</font>
			</property>
			<property name="styleSheet">
				<string notr="true">
					QToolButton {
					background-color: transparent;
					border: none;
					color: rgb(32, 137, 162);
					}

					QToolButton::hover {
					background-color: rgb(230, 237, 243);
					border: 1px solid rgb(180, 187, 194);
					border-radius: 8px;
					}
				</string>
			</property>
			<property name="text">
				<string>Ngoại Lệ</string>
			</property>
			<property name="icon">
				<iconset>
					<normaloff>../../Images/exception.png</normaloff>../../Images/exception.png
				</iconset>
			</property>
			<property name="iconSize">
				<size>
					<width>80</width>
					<height>80</height>
				</size>
			</property>
		</widget>
		<widget class="QToolButton" name="reportButton">
			<property name="geometry">
				<rect>
					<x>100</x>
					<y>210</y>
					<width>151</width>
					<height>121</height>
				</rect>
			</property>
			<property name="font">
				<font>
					<pointsize>12</pointsize>
				</font>
			</property>
			<property name="styleSheet">
				<string notr="true">
					QToolButton {
					background-color: transparent;
					border: none;
					color: rgb(32, 137, 162);
					}

					QToolButton::hover {
					background-color: rgb(230, 237, 243);
					border: 1px solid rgb(180, 187, 194);
					border-radius: 8px;
					}
				</string>
			</property>
			<property name="text">
				<string>Báo Cáo</string>
			</property>
			<property name="icon">
				<iconset>
					<normaloff>../../Images/report.png</normaloff>../../Images/report.png
				</iconset>
			</property>
			<property name="iconSize">
				<size>
					<width>55</width>
					<height>55</height>
				</size>
			</property>
		</widget>
		<widget class="QToolButton" name="toolButton">
			<property name="geometry">
				<rect>
					<x>440</x>
					<y>210</y>
					<width>151</width>
					<height>121</height>
				</rect>
			</property>
			<property name="font">
				<font>
					<pointsize>12</pointsize>
				</font>
			</property>
			<property name="styleSheet">
				<string notr="true">
					QToolButton {
					background-color: transparent;
					border: none;
					color: rgb(32, 137, 162);
					}

					QToolButton::hover {
					background-color: rgb(230, 237, 243);
					border: 1px solid rgb(180, 187, 194);
					border-radius: 8px;
					}
				</string>
			</property>
			<property name="text">
				<string>Công Cụ</string>
			</property>
			<property name="icon">
				<iconset>
					<normaloff>../../Images/tool.png</normaloff>../../Images/tool.png
				</iconset>
			</property>
			<property name="iconSize">
				<size>
					<width>55</width>
					<height>55</height>
				</size>
			</property>
		</widget>
		<widget class="QToolButton" name="notificationButton">
			<property name="geometry">
				<rect>
					<x>270</x>
					<y>210</y>
					<width>151</width>
					<height>121</height>
				</rect>
			</property>
			<property name="font">
				<font>
					<pointsize>12</pointsize>
				</font>
			</property>
			<property name="styleSheet">
				<string notr="true">
					QToolButton {
					background-color: transparent;
					border: none;
					color: rgb(32, 137, 162);
					}

					QToolButton::hover {
					background-color: rgb(230, 237, 243);
					border: 1px solid rgb(180, 187, 194);
					border-radius: 8px;
					}
				</string>
			</property>
			<property name="text">
				<string>Bảng Thông Báo</string>
			</property>
			<property name="icon">
				<iconset>
					<normaloff>../../Images/notification.png</normaloff>../../Images/notification.png
				</iconset>
			</property>
			<property name="iconSize">
				<size>
					<width>55</width>
					<height>55</height>
				</size>
			</property>
		</widget>
	</widget>
	<layoutdefault spacing="6" margin="11"/>
	<resources/>
	<connections/>
</ui>
